src/main/java/frc/robot/SwerveModule.java

```diff
@@ -89,16 +84,16 @@ public SwerveModuleState getState() {
         return new SwerveModuleState(getTransVelocity(), Rotation2d.fromRadians(getRotPosition()));
```

Comment:
A defect here? the call to getRotPosition() expects the return value in radians. 
However if you trace the call chain, you will see getRotPositionRaw()
getRotPosition() calls getRotPositionRaw() calls mRotEncoder.getAbsolutePosition() which return degrees

Reply:
I remember now. We lost couple of days during summer because we manually used CTRE tool to configure 4 CANCoders.   Some swerve modules were set correctly but different settings were set for other wheels. The problem solved when I asked Ankit to consistently checked for all 4 modules.  Change from manual CTRE tool to code now to avoid the repeat of the problem. It is less likely to have problem with the code approach because we run the same code for all modules. The benefit is realized if any of the CAN Coders is broken and requires replacement, or this code is used to run a new robot.

Reply:
We can make this change in another PR. We lost time during the summer not because the units were inconsistent, but because of the electrical/mechanical issues, along with the code approach making the units inconsistent as well, which is why it took time. It will take some time to figure it out, and for now it isn't that difficult to reset a CANCoder since if any of them break we will have to set it up anyways to set its id.

Reply:
See below